dali/pipeline/operators/python_function/python_function.cc

```diff
@@ -34,7 +35,8 @@ DALI_SCHEMA(PythonFunction)
         .AddArg("function",
                 R"code(Function object consuming and producing a single numpy array)code",
                 DALI_PYTHON_OBJECT)
-        .AddOptionalArg("num_outputs", R"code(Number of outputs)code", 1);
+        .AddOptionalArg("num_outputs", R"code(Number of outputs)code", 1)
+        .NoPrune();
```

Comment:
Why make this by default?

Reply:
My idea was that python operator always can't be assumed to have no side effects so we should not prune it by default.